person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, age
+  attr_writer id
```

Comment:
- [ ] id should only be able to be retrieved and not modified so only the Getter is defined. You can refer to [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md) for a refresher on getter and setter methods.

Reply:
Thank you for your feedback